integration-tests/fake_spine/fake_spine/vnp_spine_message_builder.py

```diff
@@ -0,0 +1,21 @@
+
+import os
+from pathlib import Path
+
+from fake_spine.pystache_message_builder import PystacheMessageBuilder
+
+TEMPLATES_DIR = "vnp_responses"
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Seen this across a few files, should this be imported from another class?

Reply:
Good spot it is done this way in some other modules

Reply:
Done!